### PR TITLE
Reduction of final Docker image size by 48%

### PR DESCRIPTION
**Pull request by gaby**
* Replaces **opendjk:13-jdk-slim-buster** image with **AdoptOpenJDK** variants.
* Reduces size of final image from **1.07GB** to **534MB** a reduction of **48%**
* Final image only contains: jre, nmap, ncrack and the built jar files.

This PR complements #75 by reducing the size of the image being published.

Build process now uses: adoptopenjdk/openjdk13:debianslim
Final image uses: adoptopenjdk/openjdk13:debianslim-jre

### CODE REVIEW

**Comment by gaby** [comment]
I had one question, would it be possible for tsunami to run with OpenJDK 14 or 15? Those images are updated more often than 13.

**Comment by gaby** [comment]
@erikvarga @magl0 Feedback on this would be appreciated.

**Comment by magl0** [comment]
>  possible for tsunami to run with OpenJDK 14 or 15?

I think so, but this was not tested so I'd like to perform some testing first before switching. And ideally we should stick with a LTS release of Java in the future.

**Comment by magl0** [comment]
PR pending internal review.

File: Dockerfile [hunk 1]
```diff
@@ -15,20 +14,28 @@ WORKDIR /usr/tsunami/repos/tsunami-security-scanner-plugins/google
...
+    && mkdir logs/
 
 WORKDIR /usr/tsunami
 
-RUN mkdir logs/
+COPY --from=0 /usr/tsunami /usr/tsunami
 
 ENTRYPOINT ["java", "-cp", "tsunami.jar:plugins/*", "-Dtsunami-config.location=tsunami.yaml", "com.google.tsunami.main.cli.TsunamiCli"]
-CMD ["--ip-v4-target=127.0.0.1", "--scan-results-local-output-format=JSON", "--scan-results-local-output-filename=logs/tsunami-output.json"]
+CMD ["--ip-v4-target=127.0.0.1", "--scan-results-local-output-format=JSON", "--scan-results-local-output-filename=logs/tsunami-output.json"]
```
Comment (magl0) [inline Dockerfile:41]: nitpick: please add a terminating newline at the end of this file.

**Pull request merged**